committee-steering/governance/community-health-check.md

```diff
@@ -0,0 +1,116 @@
+- Do you enjoy the role? What do you find challenging? How can we/is there
+something we can provide that would better support you? Do you want to continue
+or find a replacement?
+- Do you have goals for the group? If you feel that you aren’t ready to pass the
+baton, what would you like to accomplish before you do?
+
+#### Special Interest Groups:
+- How are you doing with operational tasks in [yourgroup]-governance.md?
+- Is your README accurate? have a CONTRIBUTING.md file?
+- All subprojects correctly mapped and listed in sigs.yaml?
+- Are all subproject maintained? Are there any that are being retired?
+- Tell us about your meetings: large/small, active/quiet? learnings? up to date
+on the Kubernetes community YouTube under a community group playlist? Meeting
+notes up to date?
+- When was your last public community update? (provide link)
+
+##### Membership
+- Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+- How do you measure membership? By mailing list members, OWNERs, or something
+else?
+- How does the group measure reviewer and approver bandwidth? Do you need help in
+any area now? What are you doing about it?
+- Is there a healthy onboarding and growth path for contributors in your SIG? What
+ are some activities that the group does to encourage this?What programs are you
+  participating in to grow contributors throughout the contributor ladder?
+- What programs do you participate in for new contributors?
+
+##### Current initiatives and project health
+- What are initiatives that should be highlighted, lauded, shout outs, that
+your group is proud of? Currently underway? What are some of the longer tail
+projects that your group is working on?
+- What areas does the group need the most help with?
+- Whats the average open days of a PR and Issue in your group? / what metrics does
+ your group care about and/or measure?  
+
+How can Steering help? Do you have feedback for us? Suggestions for what we
+should work on?  
+
+### Working Groups:
+Working Group Chairs should be reporting this information to their sponsoring
+SIG Chairs to be presented with the SIG.  WG Chairs should still complete the
+'You and Your Role' section and follow directions there.  
+- What was the initial mission of the group and if it's changed, how?
+- What’s the current roadmap until completion?
+- Have you produced any artifacts, reports, white papers to date?
+- Is the group active? healthy? contributors from multiple companies and/or end
+user companies?
+- Is everything in your readme accurate? posting meetings on youtube?
+- Do you have regular check-ins with your sponsoring SIGs?
+
+
+[SIG]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
+[WG]: https://git.k8s.io/community/committee-steering/governance/wg-governance.md
+[UG]: https://git.k8s.io/community/committee-steering/governance/ug-governance.md
+[governance]: https://git.k8s.io/community/governance.md
```

Comment:
this should link to https://git.k8s.io/community/governance.md#Community-Group-Annual-Health-Checks